CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### 2.6.1
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.5.2](https://fluentbit.io/announcements/v1.5.2/)
+* Amazon CloudWatch Logs for Fluent Bit 1.3.2
+* Amazon Kinesis Streams for Fluent Bit 1.5.1
+* Amazon Kinesis Firehose for Fluent Bit 1.4.1
+
```

Comment:
Is there anything worth adding in the normal "Compared to `2.x.x` this release adds:"? We usually add that section to call out any changes in the plugins customers might find interesting.

Reply:
Not really. Basically we removed printing out redundant newline characters from all plugins and update their dependencies.